activity_browser/settings.py

```diff
@@ -21,9 +20,9 @@ class BaseSettings(object):
     """Base Class for handling JSON settings files."""
 
     def __init__(self, directory: str, filename: str = None):
-        self.data_dir = directory
-        self.filename = filename or "default_settings.json"
-        self.settings_file = os.path.join(self.data_dir, self.filename)
+        self.data_dir: str = directory
```

Comment:
Should this also be an instance of `pathlib.Path`? We found it really convenient to have everything in `bw2data` be a `Path`, but maybe that is more trouble than it is worth here.